tests/Main.hs

```diff
@@ -717,6 +717,7 @@ runClashTest = defaultMain $ clashTestRoot
         , runTest "T2342A" def{hdlSim=[]}
         , runTest "T2342B" def{hdlSim=[]}
         , runTest "T2360" def{hdlSim=[],clashFlags=["-fclash-force-undefined=0"]}
+        , outputTest "T2242" def{hdlTargets=[Verilog], clashFlags=["-fclash-sanitise-netlist-ids","-fclash-hdlsyn","vivado"]}
```

Comment:
The solution we picked didn't end up being vivado-specific, we can remove `-fclash-hdlsyn vivado` from the options

Reply:
We can probably also test all `hdlTargets`, i.e. remove `hdlTargets` from here and just use the default. Since our fix isn't Verilog-specific either we would expect VHDL to also work this way. You would need to add `mainVHDL` and `mainSystemVerilog` to the test

Reply:
A raft of cleanups of the issues you pointed out awaits further review, whenever you might have time. This one in particular